network/common/src/main/java/org/apache/spark/network/sasl/SparkSaslServer.java

```diff
@@ -60,13 +60,19 @@
   static final String DIGEST = "DIGEST-MD5";
 
   /**
-   * The quality of protection is just "auth". This means that we are doing
-   * authentication only, we are not supporting integrity or privacy protection of the
-   * communication channel after authentication. This could be changed to be configurable
-   * in the future.
+   * QOP value that includes encryption.
+   */
+  static final String QOP_AUTH_CONF = "auth-conf";
+
+  /**
+   * QOP value that does not include encryption.
+   */
+  static final String QOP_AUTH = "auth";
+
+  /**
+   * Common SASL config properties for both client and server.
    */
   static final Map<String, String> SASL_PROPS = ImmutableMap.<String, String>builder()
-    .put(Sasl.QOP, "auth")
     .put(Sasl.SERVER_AUTH, "true")
```

Comment:
I don't think it applies to the client. I'm also not sure whether it's needed at all, but I'll change the code so it's only set for the server.